subsys/nrf_security/src/drivers/cracen/psa_driver.Kconfig

```diff
@@ -1737,7 +1737,7 @@ config PSA_NEED_CRACEN_KEY_DERIVATION_DRIVER
 
 config PSA_NEED_CRACEN_KMU_DRIVER
 	bool
-	default y
+	default y if !BUILD_WITH_TFM
```

Comment:
Let's add a link to the jira for KMU support so the developer doing that jira can easily find all places that need patching.

(and also make it clear that this is a temporary patch)

Reply:
Sure, I added it.